### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by josepedrocorreia** [oidc-wp/openid-connect-generic#354] [Q&A]
Hi everyone,

I'm using this plugin on a website to allow users to login via a Keycloak instance. I love it not only for being free and open-source, but also for being simple and straight to the point.

However, since we upgraded Wordpress recently, we started having some strange issues.

I'm wondering: is this project still under active development and maintenance?

I need to know if it's worth it to wait for developments or if I should just start looking for an alternative.

Cheers,
Pedro

**Comment by timnolte** [comment]
Yes, the project is still active. I am personally using the plugin on the latest WordPress release and have not had any reports of issues.

**Reply by josepedrocorreia** [reply]
Hi Tim, thanks for the quick reply! What OAuth provider are you using on your website? Maybe the issue is specific to Keycloak.

**Comment by timnolte** [comment]
Have you opened up an Issue on GitHub yet to report the problem you are having?

**Reply by josepedrocorreia** [reply]
I haven't opened an issue yet, since I'm not 100% sure what the problem might be. However, I did comment on [this PR](https://github.com/oidc-wp/openid-connect-generic/pull/349) that might be related.
